authentik/stages/authenticator_email/migrations/0001_initial.py

```diff
@@ -0,0 +1,133 @@
+# Generated by Django 5.0.10 on 2025-01-10 04:55
```

Comment:
Just here as a reminder to regen the migrations before merging

Reply:
done 👍